### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Generated by https://github.com/jsoref/spelling `f`; to maintain your repo, please consider `fchurn`

Note: I understand that the commit message style doesn't quite match. I can squish/rewrite as requested.

There are a couple of commits where I'm guessing about the corrections...

### CODE REVIEW

File: check/main.c [hunk 1]
```diff
@@ -7851,7 +7851,7 @@ static int check_chunk_refs(struct chunk_record *chunk_rec,
 			    dev_extent_rec->length != length) {
 				if (!silent)
 					fprintf(stderr,
-						"Chunk[%llu, %u, %llu] stripe[%llu, %llu] dismatch dev extent[%llu, %llu, %llu]\n",
+						"Chunk[%llu, %u, %llu] stripe[%llu, %llu] mismatch dev extent[%llu, %llu, %llu]\n",
```
Comment (jsoref) [inline check/main.c]: I'm not sure about this

File: check/mode-lowmem.c [hunk 2]
```diff
@@ -2636,7 +2636,7 @@ static int process_one_leaf(struct btrfs_root *root, struct btrfs_path *path,
 	if (err & LAST_ITEM)
 		goto out;
 
-	/* still have inode items in thie leaf */
+	/* still have inode items in the leaf */
```
Comment (jsoref) [inline check/mode-lowmem.c]: Possibly "this"?

File: convert/common.c [hunk 3]
```diff
@@ -76,7 +76,7 @@ static inline int write_temp_super(int fd, struct btrfs_super_block *sb,
 }
 
 /*
- * Setup temporary superblock at cfg->super_bynter
+ * Setup temporary superblock at cfg->super_bytenr
```
Comment (jsoref) [inline convert/common.c]: Pretty sure this is right...

File: send-stream.c [hunk 4]
```diff
@@ -35,7 +35,7 @@ struct btrfs_send_stream {
 
 	/*
 	 * end of last successful read, equivalent to start of current
-	 * malformated part of block
+	 * malformed part of block
```
Comment (jsoref) [inline send-stream.c]: this might be objectionable...

File: show-blocks [hunk 5]
```diff
@@ -33,15 +33,15 @@ rcParams['backend'] = 'Agg'
 rcParams['interactive'] = 'False'
 from pylab import *
 
-class AnnoteFinder:
+class AnnotationFinder:
```
Comment (jsoref) [inline show-blocks]: It's possible that `annote` is intentional -- I can easily drop any individual commits

File: tests/fsck-tests/025-file-extents/test.sh [hunk 6]
```diff
@@ -15,7 +15,7 @@ prepare_test_dev 128M
 
 # Do some write into a large prealloc range
 # Lowmem mode can report missing csum due to wrong csum range
-test_paritical_write_into_prealloc()
+test_partial_write_into_prealloc()
```
Comment (jsoref) [inline tests/fsck-tests/025-file-extents/test.sh]: Not sure about this

File: check/main.c (hunk 1 above)
Comment (kdave) [reply check/main.c]: It's 'mismatch'

File: check/mode-lowmem.c (hunk 2 above)
Comment (kdave) [reply check/mode-lowmem.c]: Yes, 'this'

File: convert/common.c (hunk 3 above)
Comment (kdave) [reply convert/common.c]: Indeed, refers to the member 'super_bytenr' in struct btrfs_fs_info

File: send-stream.c (hunk 4 above)
Comment (kdave) [reply send-stream.c]: I don't remember exactly, as the stream is a formatted stream of bytes, the 'malformatted' might make sense too, but 'malformed' is not cofusing in this context

File: show-blocks (hunk 5 above)
Comment (kdave) [reply show-blocks]: The 'show-blocks' script seems to be stale, I don't remember anyone using it, so fixing there is not necessary as we might delete it anyway.

File: tests/fsck-tests/025-file-extents/test.sh (hunk 6 above)
Comment (kdave) [reply tests/fsck-tests/025-file-extents/test.sh]: 'partial' looks correct to me

**Comment by kdave** [comment]
Thanks for the fixes, no matter how much we try not to introduce them, they keep coming. The changes would be better grouped logically, eg. documentation, CHANGES, and code. I'll do that as the separate commits make it easy to cherry-pick and squash relevant changes, so if you have further updates, please keep it as is and I'll do the final merge.

**Comment by jsoref** [comment]
@kdave : Thanks for offering to rearrange this PR. At present, I don't have any other changes -- my script ran through the entire corpus.

**Comment by kdave** [comment]
Changes grouped into commits, now pushed to devel branch. Closing.

**Pull request closed without merging**